rules/no-abstract-functions.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @fileoverview Discourage use of abstract functions
+ * @author Nicolas Feignon <nfeignon@gmail.com>
+ */
+
+'use strict';
+
+module.exports = {
+
+    meta: {
+
+        docs: {
+            recommended: true,
+            type: 'error',
+            description: 'Discourage use of abstract functions',
+        },
+
+        schema: []
+
+    },
+
+    create: function (context) {
+
+        function inspectFunctionDeclaration(emitted) {
+            var node = emitted.node;
+            if (!emitted.exit && node.is_abstract) {
+                context.report({
+                    node: node,
+                    message: 'Avoid using abstract functions'
```

Comment:
change message to `"<FUNCTION NAME>": Avoid using abstract functions`. Func name can be accessed from `node` object.